class/defaults.yml

```
@@ -117,16 +117,13 @@ parameters:
       synchronize: true
       limits:
         "Container":
-          max:
-            cpu: "1"
-            memory: "4Gi"
           min:
             cpu: "10m"
             memory: "4Mi"
             ephemeral-storage: "100Ki"
           default:
-            cpu: "200m"
-            memory: "200Mi"
+            cpu: "1"
```

I've rarely seen apps or charts setting a limit to 0.5 or higher. A high-demanding workload usually has resources defined, so a really-high default limit is probably not needed
```suggestion
            cpu: "600m"
```